src/js/modules/SectionHeaderSerif.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const SectionHeaderSerif = ({ title }) => (
```

Comment:
Yea, I noodled on this. I thought the sass was sufficiently different (since the section header is sized differently, has an underline thing and has an arrow option), that I made a new component so it wouldn't seem like the two were related. 
![screen shot 2018-04-06 at 12 00 51 pm](https://user-images.githubusercontent.com/3039125/38433943-4734aaba-3992-11e8-9701-dd2983515c65.png)

Reply:
Open to mashing them together tho -- let me think on this. I can make the underline an option as well.

Reply:
Updated! Thanks for the feedback on this.